src/test/HostActivation.Tests/FrameworkResolution/IncludedFrameworksSettings.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.Cli.Build;
+using Microsoft.DotNet.Cli.Build.Framework;
+using Xunit;
+
+namespace Microsoft.DotNet.CoreSetup.Test.HostActivation.FrameworkResolution
```

Comment:
`FrameworkResolution` seems like a weird place for this, since it is not really doing any resolving.

Reply:
But I guess we don't really have general parsing tests, so this might be the closest right now and already has infrastructure to reuse.

Reply:
Well - I agree that ideally runtimeconfig parsing would be somewhere separate, but currently we only have tests related to frameworks in this area. And the `includedFrameworks` are effectively a framework resolution feature.

But you're right, mostly it was about existing infra.